rockcraft/extensions/expressjs.py

```diff
@@ -234,7 +234,7 @@ def _app_package_json(self) -> dict[str, Any]:
                 )
         except json.JSONDecodeError as exc:
             raise ExtensionError(
-                "failed to parse package.json file",
+                "failed to parse package.json file, make sure it is a valid JSON",
```

Comment:
```suggestion
                "failed to parse package.json; it might contain invalid JSON",
```

I don't think we normally mix the remedy into the error.